api/src/feeds/impl/feeds_api_impl.py

```diff
@@ -12,6 +16,24 @@
 from feeds_gen.models.latest_dataset import LatestDataset
 from feeds_gen.models.source_info import SourceInfo
 
+from database.database import Database
+
+db = Database()
```

Comment:
Nice to have: we should probably set up a mechanism to have a single instance of the database (singleton pattern). Instantiating the database as a global variable might increase the risk of concurrency issues and creates a lack of control over initialization in case we need to make any changes to it.

Reply:
moved to a global variable in database module